### PR TITLE
Toolsets

### PR DESCRIPTION
**Pull request by ch40s3k**

### CODE REVIEW

File: alamo_worker/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__author__ = 'hubert.sajnaj'
```
Comment (deejay1) [inline alamo_worker/__init__.py]: drop that

**Comment by deejay1** [comment]
Also a README.md file is missing here, LICENSE is missing

File: tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+__author__ = 'hubert.sajnaj'
```
Comment (deejay1) [inline tests/__init__.py]: and also drop that

Comment (ch40s3k) [inline tests/__init__.py]: Done

File: alamo_worker/__init__.py (hunk 1 above)
Comment (ch40s3k) [inline alamo_worker/__init__.py]: Done

**Comment by deejay1** [comment]
:+1:

**Comment by pnuckowski** [comment]
+

**Pull request merged**